x-pack/legacy/plugins/siem/public/components/ml/influencers/get_network_from_influencers.ts

```diff
@@ -7,12 +7,18 @@
 import { DestinationOrSource } from '../types';
 
 export const getNetworkFromInfluencers = (
-  influencers: Array<Record<string, string>>
+  influencers: Array<Record<string, string>>,
+  ip?: string
 ): { ip: string; type: DestinationOrSource } | null => {
   const recordFound = influencers.find(influencer => {
     const influencerName = Object.keys(influencer)[0];
+    const influencerValue = Object.values(influencer)[0];
```

Comment:
nit - `Object.entries` will give you key and value at the same time

Reply:
I created a `getEntries` function which normalizes this data and optionally returns null for their values which makes all of this logic cleaner and within that function I use the `Object.entries`.

Thanks for the advice and keeping me honest to the ways of the JavaScript